bunq/sdk/client.py

```diff
@@ -15,6 +15,16 @@ class ApiClient(object):
     :type _api_context: bunq.sdk.context.ApiContext
     """
 
+    # Endpoints not requiring active session for the request to succeed.
+    _URL_DEVICE_SERVER = 'device-server'
+    _URI_INSTALLATION = 'installation'
+    _URI_SESSION_SERVER = 'session-server'
```

Comment:
Please put these constants after the map (just like we do in PHP).

Reply:
That won't work in python 😅 ? 

https://stackoverflow.com/questions/1590608/is-it-possible-to-forward-declare-a-function-in-python

Reply:
Ahh, you're right! My apologies.

Reply:
python is not that magical ✨ 😂

Reply:
@OGKevin declare time mixed with run time? Tell me it is not magical :P